hype-submitter_2.11/src/main/scala/com/spotify/hype/package.scala

```diff
@@ -16,14 +16,11 @@ package object hype {
   object VolumeRequest {
     def apply(storageClass: String, mountPath: String): model.VolumeRequest =
       model.VolumeRequest.volumeRequest(storageClass, mountPath)
-
-    def apply(id: String, storageClass: String, mountPath: String): model.VolumeRequest =
-      model.VolumeRequest.volumeRequest(id, storageClass, mountPath)
   }
 
-  object ExistingVolume {
-    def apply(claimName: String): model.VolumeRequest =
-      model.VolumeRequest.existingClaim(claimName)
+  object PersistentVolume {
```

Comment:
These names could perhaps be improved on to better reflect the behaviour.

`VolumeRequest` - a temporary volume that will be deleted after the run, maybe better called `TempVolume`
`PersistentVolume` - a volume that is created and reused between runs, and has to be externally managed